### PR TITLE
DEVX-2457: Exclude helper templates from CLI results

### PR DESCRIPTION
**Pull request by roribio**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/commands/templates/discover.js [hunk 1]
```diff
@@ -120,7 +120,8 @@ class DiscoverCommand extends BaseCommand {
 
     try {
       const searchCriteria = {
-        statuses: ['Approved']
+        statuses: ['Approved'],
+        categories: ['!helper-template']
```
Comment (slitviachenko) [inline src/commands/templates/discover.js:124]: let's use a dedicated constant, you need to import it first
```
const { SEARCH_CRITERIA_FILTER_NOT } = require('@adobe/aio-lib-templates')
```
then
```suggestion
        categories: [SEARCH_CRITERIA_FILTER_NOT + 'helper-template']
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/aio-cli-plugin-app-templates",
-  "version": "1.0.0-beta.11",
+  "version": "1.0.0-beta.12",
```
Comment (slitviachenko) [inline package.json:3]: I think we can make a first release, `1.0.0` 👍

**Review by slitviachenko — CHANGES_REQUESTED** [review]
left some comments

File: src/commands/templates/discover.js (hunk 1 above)
Comment (roribio) [reply src/commands/templates/discover.js]: Done ✅

File: package.json (hunk 2 above)
Comment (roribio) [reply package.json]: Done ✅

**Review by slitviachenko — APPROVED** [review]

**Pull request merged**